bears/vcs/git/GitCommitBear.py

```diff
@@ -106,6 +106,9 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
         pos = stdout.find('\n')
         shortlog = stdout[:pos] if pos != -1 else stdout
         body = stdout[pos+1:] if pos != -1 else ''
+        if pos != -1:
+            if stdout[pos+2] == '\n':
+                yield Result(self, 'Extra Newlines found.')
 
```

Comment:
return statement missing. We want this function to terminate if more than a newline is found